### DISCUSSION TITLE
Button to re-sync with server

### DISCUSSION THREAD

**Discussion by WinterLoki** [LeoKlaus/plappa#49] [Feature Requests]
Pulling down on the list in the Books tab doesn't seem to get new information from the server. Can a re-sync button be added to force check when I add new stuff to my server?

**Comment by LeoKlaus** [comment]
Maybe. The initial sync process (the one you can trigger by pulling down in the books tab) doesn't get all metadata.
This is in parts due to limitations of the ABS/Jellyfin APIs and in parts for performance reasons.

To completely refresh the metadata for a book, you can go to the detail page of that book (the one where you can see chapters) and pull down to refresh there.
That should get all new information from the server.

New books should always automatically appear when you launch plappa or when using the pull-to-refresh-gesture in the books tab.

**Comment by WinterLoki** [comment]
The issue I was having was that I had updated a book to be part of a series but for some reason it was showing both on it's own (I have series collapsed in Books) as well as in the series. I figured it was because the info saying this book is in a series hadn't fully updated.

**Comment by mariomaz87** [comment]
Is there a way to force the resync also for podcasts?

**Reply by LeoKlaus** [reply]
Yes, it's the same as refreshing audiobooks. You can pull down to refresh in the "Podcasts" tab, the detail view for a podcast or an episode to refresh the metadata for each.

**Comment by migol** [comment]
It would be great if we had a way to "Purge the cache" - a one shot thing to fix the issues.
